src/Services/FHIR/FhirCarePlanService.php

```diff
@@ -550,7 +550,7 @@ private function getCarePlanTextFromDetails(array $details): array
 
         if (!empty($descriptions)) {
             // Properly escape HTML entities for each description
-            $escapedDescriptions = array_map('text', $descriptions);
+            $escapedDescriptions = array_map(text(...), $descriptions);
```

Comment:
I didn't realize PHP added this capability for static typing on the function, this is really nice.  Great improvement.